pom.xml

```
@@ -93,7 +92,32 @@
         <dependency>
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
-            <version>30.1.1-jre</version>
+            <version>33.3.1-jre</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.errorprone</groupId>
+            <artifactId>error_prone_annotations</artifactId>
+            <version>2.28.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib-jdk8</artifactId>
+            <version>1.9.10</version>
+        </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcprov-jdk18on</artifactId>
+            <version>1.80</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
+            <version>1.27.1</version>
+        </dependency>
```

Use a Jenkins plugin instead of including commons-compress into the plugin yourself.
```suggestion
    <dependency>
        <groupId>io.jenkins.plugins</groupId>
        <artifactId>commons-compress-api</artifactId>
    </dependency>```

---

Thanks for review.
Dep Minio relays on this dep commons-compress, this is only to resolve RequireUpperBoundDeps, I have moved the transitive dependencies to the proper section(<dependencyManagement>)